packages/cli/src/services/config-service.ts

```diff
@@ -35,23 +35,29 @@ async function compileProject(projectPath: string): Promise<void> {
 }
 
 function readProjectConfig(userProjectPath: string): Promise<BoosterConfig> {
-  // eslint-disable-next-line @typescript-eslint/no-var-requires
   const userProject = loadUserProject(userProjectPath)
   return new Promise((resolve): void => {
-    const app: BoosterApp = userProject.Booster
+    const app = userProject.Booster
     app.configureCurrentEnv((config: BoosterConfig): void => {
       checkEnvironmentWasConfigured(app)
       resolve(config)
     })
   })
 }
 
-function loadUserProject(userProjectPath: string): { Booster: BoosterApp } {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function loadUserProject(userProjectPath: string): { Booster: any } {
+  /* TODO: The main goal of loading the project here is
+   * to get the configuration. In future releases, we'll provide
+   * a static version of the config that can be loaded without
+   * dynamically requiring the user project from the CLI.
+   */
   const projectIndexJSPath = path.resolve(path.join(userProjectPath, 'dist', 'index.js'))
   return require(projectIndexJSPath)
 }
 
-function checkEnvironmentWasConfigured(app: BoosterApp): void {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function checkEnvironmentWasConfigured(app: any): void {
```

Comment:
Why `any`?

Reply:
I have the same concern and the reasons are the same as the previous comments. Now, all the usage of `app` inside this function is not compile-time checked

Reply:
The goal is to get rid of dynamic project requires, read more about it here: https://github.com/boostercloud/booster/pull/612#issuecomment-789051941